Cargo.toml

```diff
@@ -24,6 +24,7 @@ log = "0.3"
 env_logger = "0.3"
 toml = { version = "0.2", features = ["serde"] }
 open = "1.1"
+regex = "0.1.80"
```

Comment:
the only thing it might cause is two version of regex, given that iirc this is a dependency of something.

I could also just update all deps; https://github.com/azerupi/mdBook/pull/206 is doing some of that